tests/unit/services/ajax-test.js

```diff
@@ -134,6 +134,49 @@ test("request() promise label is correct", function(assert) {
   assert.equal(postPromise._label, 'ember-ajax: POST to /posts');
 });
 
+test("post() promise label is correct", function(assert) {
+  service = Service.create();
+  const url = '/posts';
```

Comment:
No, I specifically made no reference to immutability. Local vars that "happen" to not change, are not `const`, they are let. `const` is meant to convey, please do to rebind this given variable.